ucx.spec.in

```diff
@@ -1,98 +1,67 @@
 %global rel @RPM_RELEASE@
 %global version @VERSION@
-%global pkgname @PACKAGE@
 %global __check_files %{nil}
 %global debug_package %{nil}
 %bcond_with valgrind
-%global _binary_filedigest_algorithm 1
-%global _source_filedigest_algorithm 1
 
-%global lt_release @LT_RELEASE@
-%global lt_version @LT_CURRENT@.@LT_REVISION@.@LT_AGE@
+%{!?configure_options: %global configure_options %{nil}}
 
-%{!?configure_options: %define configure_options %{nil}}
-
-Name: %{pkgname}
-Summary: Unified Communication X
+Name: ucx
 Version: %{version}
 Release: %{rel}
+Summary: Unified Communication X
 
-License: BSDv3
-Group: Applications
-Source: %{pkgname}-%{version}.tar.gz
-
-BuildRoot: %(mktemp -ud %{_tmppath}/%{name}-%{version}-%{release}-XXXXXX)
+License: BSD
 URL: http://www.openucx.org
-Provides: ucx
-Packager: openucx
-Vendor: openucx
+Source: %{name}-%{version}.tar.gz
+
+Requires(post): /sbin/ldconfig
+Requires(postun): /sbin/ldconfig
 
+BuildRequires: numactl-devel
+BuildRequires: automake autoconf libtool
 
 %description
 UCX is a communication library implementing high-performance messaging.
 
+%package devel
+Summary: Header files and libraries required to develop with ucx
+Requires: %{name}%{?_isa} = %{version}-%{release}
+
+%description devel
+UCX is a communication library implementing high-performance messaging.
+Requires either RDMA-capable device, or Cray Gemini, for inter-node
```

Comment:
You may want to mention intra-node communication support with various flavors of shared memory mechanism, including sysv, cma, knem, xpmem, etc.